docs/manual/grelfunctions.md

```diff
@@ -591,6 +591,9 @@ Returns a boolean indicating whether o has a member field called [name](expressi
 ###### coalesce(o1, o2, o3, ...) {#coalesceo1-o2-o3-}
 Returns the first non-null from a series of objects. For example, `coalesce(value, "")` would return an empty string “” if `value` was null, but otherwise return `value`.
 
+###### levenshteinDistance(s1, s2) {#levenshteinDistance-s1-s2}
```

Comment:
I think this should go in the `String functions` section of this documentation. I would suggest directly after the `length` function

Reply:
Done.